### ISSUE TITLE
Pano panel not showing with window tiling activated on pop-shell

### ISSUE THREAD

**Issue by Avijneyam** [pop-os/shell#1534]
**(1) Issue/Bug Description:**
Panel not showing with window tiling activated on pop-shell

**(2) Steps to reproduce (if you know):**
enable tiling in pop-shell
This [pano](https://github.com/oae/gnome-shell-pano) extention is not working

**(3) Expected behavior:**
pano panel should show after enabling pop-shell too

**(4) Distribution (run `cat /etc/os-release`):**
 
```
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2022.3"
VERSION_ID="2022.3"
VERSION_CODENAME="kali-rolling"
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="https://www.kali.org/"
SUPPORT_URL="https://forums.kali.org/"
BUG_REPORT_URL="https://bugs.kali.org/"
```

**(5) Gnome Shell version:**
GNOME Shell 43.0

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**

886a069c0582b371e90ac3602b1747ea5fba616c

**(7) Where was Pop Shell installed from:**

Github

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**

2x Primary(1080p), Secondary(720p) both are Horizontal

**(9) Other Installed/Enabled Extensions:**
pano

**(10) Other Notes:**

**Comment by isamert** [comment]
This seems to be a more general problem with extensions showing windows/frames. This [search-light](https://github.com/icedman/search-light) extension is also not working with pop-shell in tiled mode. Although it works if there aren't any windows present on the workspace (this is also the case for pano extension).

**Comment by Avijneyam** [comment]
> This seems to be a more general problem with extensions showing windows/frames. This [search-light](https://github.com/icedman/search-light) extension is also not working with pop-shell in tiled mode. Although it works if there aren't any windows present on the workspace (this is also the case for pano extension).

yes same happening with pano extension... if pop-shell has option to manually add ignore words/extensions in settings then it would be good so user can add to ignore others extensions

**Comment by data-flux** [comment]
So the way exceptions currently work means they need a WM_CLASS and WM_TITLE, and Search-Light and Pano have neither.
`xprop` on either of them yields:
`XdndProxy(WINDOW): window id # 0x20000b`
(the same result as xprop on the task bar)

Pop OS gets these values through Meta.Window.get_title() and Meta.Window.get_wm_class()
However, the popups provided by these extensions do not get listed in the Meta windows. 

Maybe that will give someone more to work with, I'm not familiar enough with gjs to do anything with that, unfortunately.

**Comment by durka** [comment]
> Although it works if there aren't any windows present on the workspace

Note: I found it also works if a window on a different monitor is focused.
